proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
```

Comment:
I guess folks here are OK with representation of counter with-respect-to default or uninitialized representation ... 0 for all fields on Read, which agrees with not setting the counter_data() field on Write (since 0 is default for proto3).

For meter, default behavior of tagging all packets GREEN makes sense. As for how to represent it, 0 for all fields doesn't make sense to me (since that says the rate is zero => RED). What do folks here think about using "-1" for all fields to indicate "no rate limiting" (i.e. all packets GREEN, which is the default behavior).
All fields in MeterConfig are int64, so -1 can be proposed as the "default" state.
What do you think?